### DISCUSSION TITLE
Ability to change time before hover activates

### DISCUSSION THREAD

**Discussion by CharlieDixon** [rxhanson/Charmstone-Community#6] [Ideas]
I think it takes around 1-1.5 seconds for the switch to happen when hovering over an icon. My personal preference would be for it to be a little faster. Some sort of slider to customise that option would be great.

**Comment by rxhanson** [comment]
Here's the terminal command to modify it now:

`defaults write com.knollsoft.Charmstone hoverMillis -int 800`

(800 ms is the default setting).

The UI is purposefully quite slim at this time. At a time when the UI starts to grow is when I'll consider adding this as a slider.

**Reply by rxhanson** [reply]
(Restart the app after executing that command)

**Reply by CharlieDixon** [reply]
Thank you for this
